webapp/portlet/src/main/webapp/vue-apps/common/initComponents.js

```diff
@@ -58,6 +59,7 @@ const components = {
   'unread-badge': UnreadBadge,
   'alert-notifications': Notifications,
   'ripple-hover-button': RippleHoverButton,
+  'attach-image-draggable-zone': AttachImageDraggableZone,
```

Comment:
Use more generic name for this zone that can be used for other attachment types

```suggestion
  'attachments-draggable-zone': AttachmentsDraggableZone,
```

Reply:
done